.github/workflows/publish-latest.yml

```diff
@@ -61,6 +61,7 @@ jobs:
     runs-on: ubuntu-latest
     environment: deployment
     needs:
+      - setup
```

Comment:
optional, adding this in to remove some warnings

Reply:
Which warnings?